consent_manager/js_api/introduction.rst

```diff
@@ -0,0 +1,23 @@
+Introduction
+------------
+
+Consent Manager provide JavaScript API that allows the user to:
+
+    * Get compliance types
+    * Get new compliance types
+    * Set initial compliance settings
+    * Set compliance settings
+    * Get compliance settings
+    * Send data subject request
+
+JavaScript API is implemented by providing global JavaScript objects queue responsible for executing command:
+
+.. function:: ppms.cm.api(command, ...args)
+.. function:: dataLayer.push({event: command, ...args})
+
+`dataLayer.push` interface is only for backward compatibility and you can read more about this particular case below. We recommend ```ppms.cm.api```.
```

Comment:
Too many quotes or is this intended?
![2019-08-21T171559](https://user-images.githubusercontent.com/4488774/63444687-621fb000-c437-11e9-83ff-033c5987547a.png)

Reply:
I agree, at first glance it seems that `dataLayer.push` and `ppms.cm.api` should use the same highlighting method. Anyways, this pattern repeats in a few places across the documentation, so if this was not intended and there is a change to be made, let's make it everywhere.

Reply:
yes, fixed